src/wiiu/input.c

```diff
@@ -49,13 +107,23 @@ void wiiu_input_update(void) {
     CHECKBTN(VPAD_BUTTON_STICK_R, RS_CLK_FLAG);
     CHECKBTN(VPAD_BUTTON_PLUS,    PLAY_FLAG);
     CHECKBTN(VPAD_BUTTON_MINUS,   BACK_FLAG);
+    CHECKBTN(VPAD_BUTTON_HOME,    SPECIAL_FLAG);
 #undef CHECKBTN
 
+    static uint64_t home_pressed = 0;
+    // If the button was just pressed, reset to current time
+    if (vpad.trigger & VPAD_BUTTON_HOME) home_pressed = millis();
+    if (btns & VPAD_BUTTON_HOME && millis() - home_pressed > 3000) wiiu_error_exit("Goodbye!");
```

Comment:
You shouldn't use `wiiu_error_exit` to quit the application after the stream has been started. That will forcefully quit the application without properly disconnecting, uninitializing GX2 or freeing memory.

Instead use `WHBProcStopRunning` which will quit out of the main loop.